pipeline_steps/tempest.groovy

```diff
@@ -32,29 +32,27 @@ def tempest(){
   common.conditionalStage(
     stage_name: "Tempest Tests",
     stage: {
-      try{
-        def result = tempest_run()
-        def second_result = ""
-        if(result.contains("Race in testr accounting.")){
-          second_result = tempest_run()
-        }
-        if(second_result.contains("Race in testr accounting.")) {
-          currentBuild.result = 'FAILURE'
-        }
+      retry(2){
+        try{
+          def result = tempest_run()
+          if(result.contains("Race in testr accounting.")) {
+            currentBuild.result = 'FAILURE'
```

Comment:
It was to set it to failure out of caution in case the test results were a false positive. If the `Race in testr accounting` message is in the output, that means the results of one or more workers has been lost (if running in parallel). Tempest won't fail the build even if that error is thrown, so I had to set it manually.

https://bugs.launchpad.net/testrepository/+bug/1538941

I don't believe I've seen it happen since we started using specifically 3 workers. Additionally, the parallel runner is only used in multi-node AIO tests in branches that aren't master since stable/ocata has `--serial` hardcoded. So this is potentially not really a problem anymore. I just kept it in because I'm paranoid and nothing makes me sadder than false positives.

https://github.com/openstack/openstack-ansible-os_tempest/blob/stable/ocata/tasks/tempest_run.yml#L20